src/backend/core/api/viewsets.py

```diff
@@ -604,8 +605,19 @@ def attachment_upload(self, request, *args, **kwargs):
         extension = serializer.validated_data["expected_extension"]
         key = f"{document.key_base}/{ATTACHMENTS_FOLDER:s}/{file_id!s}.{extension:s}"
 
+        # Determine the content type of the file
+        file = serializer.validated_data["file"]
+        content_type, _ = mimetypes.guess_type(file.name)
```

Comment:
This was already done in the serializer. You don't need to do it here again : https://github.com/suitenumerique/docs/blob/main/src/backend/core/api/serializers.py#L367